editor/src/core/workers/parser-printer/__snapshots__/parser-printer.spec.ts.snap

```diff
@@ -719,6 +719,7 @@ const b = (n) => n > 0 ? n : a(10)
           "uniqueID": "",
         },
         "type": "JSX_CONDITIONAL_EXPRESSION",
+        "uid": "7dd",
         "uniqueID": "",
```

Comment:
Why is there still an empty `uniqueID` showing up?

Reply:
Nice catch, removed in e258b7323b2f1788cbedb053cf22611a3e3badb1